src/main/java/com/hubspot/imap/protocol/command/StringLiteralCommand.java

```diff
@@ -0,0 +1,22 @@
+package com.hubspot.imap.protocol.command;
+
+import java.nio.charset.Charset;
+
+import com.google.common.base.Joiner;
+
+public class StringLiteralCommand extends BaseImapCommand {
```

Comment:
should this class be abstract?

Reply:
What would be the reason for making this abstract? I see this as a command that is used to simply send a literal string to the server regardless of intention